### PR TITLE
2018

### PR DESCRIPTION
**Pull request by NathanielHorne**
Made the changes Tim and Erik suggested.

### CODE REVIEW

**Comment by ErikBoesen** [comment]
Yeah, looks like you've got a lot of merge conflicts here.

File: _pages/robot/2018.html [hunk 1]
```diff
@@ -1,44 +1,50 @@
...
-        <li><img src="/assets/img/first.svg">2017<br> North VA Innovation in Control</li>
-        <li><img src="/assets/img/first.svg">2017<br> North MD Creativity Award</li>
-        <li><img src="/assets/img/first.svg">2017<br> Chesapeake Innovation in Control</li>
-    </ul>
-    <img src="/assets/img/robots/2017.jpg" alt="The 2017 robot being carried onto the ground.">
-    <p>The 2017 game is proving to be one of the most challenging to date for our experienced team. Designed by Disney, FIRST STEAMWORKS aims to challenge teams to create mechanisms capable of delivering 11 inch diameter gears to human players stationed on the Jules Verne inspired airship. Teams also have the task of shooting 5 inch diameter wiffle balls to add fuel to the boiler, sending steam to fuel the airship. For a more detailed description of the game, see the <a href="https://www.youtube.com/watch?v=EMiNmJW7enI">the official video reveal</a>.</p>
+    <h1>2018 Robot</h1>
+    <embed width="660" height="460"
+src="https://www.youtube.com/v/jqxHeneuZwA">
+    </embed>
```
Comment (ErikBoesen) [inline _pages/robot/2018.html]: This is actually the first time I've seen the `<embed>` tag used. Given that we currently use `<iframe>`s, you should use that instead. Also, `<embed>`s have no closing tag.

File: _pages/robot/2018.html [hunk 2]
```diff
@@ -1,44 +1,50 @@
...
+    <!--TODO: add images-->
+    <p>The 2018 game's fusion of 20th century arcade gamming and 21st century robotics is possibly one of the most challenging scenario to date. For a more detailed description of the game, see the <a href="https://youtu.be/HZbdwYiCY74">the official video reveal</a>.</p>
```
Comment (ErikBoesen) [inline _pages/robot/2018.html]: scenario(s)

File: _pages/robot/2018.html [hunk 3]
```diff
@@ -1,44 +1,50 @@
...
-        <li><strong>Pneumatic Gear Picker:</strong> Pneumatic Gear Picker: Unlike most robots, we have the ability to pick up gears off the floor in an active delivery system. Using a pneumatically actuated hook we are able to quickly collect gears deposited from the delivery station and place them on the airship’s springs. This allows up to be very versatile and collect any gears left by other teams, maximizing efficiency of the alliance. </li>
-        <li><strong>Climber: </strong>Utilizing a massive gear reduction and milled slots into a rotating plate, we are able to latch onto a knot, pulling our 110 pound robot up into the scoring an onto the rising airship.</li>
+        <li><strong>Arcade Drive:</strong> Fittingly, arcade drive will be our drive method of choice this year. This allows</li>
+        <!--TODO: Add more details about the crane-->
+        <!--TODO: Add the length of the crane arm here-->
+        <li><strong>Power Cube Delivery Crane:</strong> From the begining, the focus of our robot was to deliver power cubes to the scale; however, this goal possed some very specific problems. The following mechanisms are our our solutions for these issues.</li>
```
Comment (ErikBoesen) [inline _pages/robot/2018.html]: posed

File: _pages/robot/2018.html [hunk 4]
```diff
@@ -1,44 +1,50 @@
...
-    <p>We drive using a pilot-copilot operating system with one driver controlling robot motion and the other controlling all other robot functions. This allows each driver to control their own independent task, making operating the robot more efficient.</p>
-    <p>Our vision system uses newly introduced WPILib cscore to operate three cameras and preform image processing on gear vision target to deliver gears to the airship.</p>
+    <p>As in years past, our robot's code will be written in Python, while our driverstation (a graphical user interface that communicates between the drivers and the robot) utilizes a combination of JavaScript, HTML and CSS. The framework for the driverstation can be found <a href="https://github.com/FRCDashboard/FRCDashboard">FRCDashboard</a>here.</p>
```
Comment (ErikBoesen) [inline _pages/robot/2018.html]: It's called a Dashboard, the driverstation is either a) the app FIRST provides to to help us communicate with our robot (this is the one where you can see things like the Enable button), or b) the literal touchscreen laptop.

Comment (ErikBoesen) [reply _pages/robot/2018.html]: Also please use the same language we used last year about FRCDashboard; since our team created it we should plug it by name. (Change the "last year" bit though, of course.)

File: _pages/robot/2018.html [hunk 5]
```diff
@@ -1,44 +1,50 @@
...
     <ul>
-        <li><a href="https://github.com/frc1418/2017-robot">Robot code</a></li>
-        <li><a href="https://github.com/frc1418/2017-dashboard">Dashboard code</a></li>
+      <!--TODO: change the website code here-->
+        <li><a href="https://github.com/frc1418/2018-robot">Robot code</a></li>
+        <li><a href="https://github.com/frc1418/2018-dashboard">Dashboard code</a></li>
         <li><a href="https://github.com/frc1418/VictiScout">VictiScout scouting system</a></li>
-        <li><a href="https://github.com/frc1418/vs">vs scouting data analysis system</a></li>
-        <li><a href="https://github.com/frc1418/vufine">VUFINE heads-up display code</a></li>
+        <li><a href="https://github.com/frc1418/vs">V.S. scouting data analysis system</a></li>
```
Comment (ErikBoesen) [inline _pages/robot/2018.html]: Just call it `vs` not V.S.

**Comment by ErikBoesen** [comment]
Good work! Just fix what I mentioned above and I'll merge.

File: _pages/robot/2018.html (hunk 4 above)
Comment (NathanielHorne) [reply _pages/robot/2018.html]: Okay.

File: _pages/robot/2018.html [hunk 6]
```diff
@@ -1,44 +1,48 @@
...
-        <li><strong>Pneumatic Gear Picker:</strong> Pneumatic Gear Picker: Unlike most robots, we have the ability to pick up gears off the floor in an active delivery system. Using a pneumatically actuated hook we are able to quickly collect gears deposited from the delivery station and place them on the airship’s springs. This allows up to be very versatile and collect any gears left by other teams, maximizing efficiency of the alliance. </li>
-        <li><strong>Climber: </strong>Utilizing a massive gear reduction and milled slots into a rotating plate, we are able to latch onto a knot, pulling our 110 pound robot up into the scoring an onto the rising airship.</li>
+        <li><strong>Arcade Drive:</strong> Fittingly, arcade drive will be our drive method of choice this year. This allows</li>
```
Comment (twinters007) [inline _pages/robot/2018.html]: Arcade drive is our user input mode, while the chassis is a tank drive configuration

Comment (twinters007) [reply _pages/robot/2018.html]: Just making sure you’re saying what you want to

**Comment by NathanielHorne** [comment]
Alright. The changes have been made. I submitted a new pr as well.

File: _pages/robot/2018.html (hunk 6 above)
Comment (NathanielHorne) [reply _pages/robot/2018.html]: Yup. Don't worry. I'll clarify that though.

File: index.html [hunk 7]
```diff
@@ -22,19 +22,20 @@
...
+        <p>We are team 1418 Vae Victis, a <a href="http://firstinspires.org/robotics/frc">FIRST Robotics Competition (FRC)</a> from George Mason High School in Falls Church, VA. Since our founding in 2004, our team has grown in every way, and has maintained a high place in competition and in the community. To do this, we maintain an active and engaged student body of around fifteen students who work with the team on and off season.</p>
+        <p></p>
         <p>We are proud to have experienced success in competition. We have made it to the FIRST World Championship in St. Louis, Missouri for the past two years, and plan to continue our streak. Outside of competition, we pride ourselves on involvement in our community. We have hosted free summer camps to educate elementary school students on basic LEGO Robotics, and volunteer to help run FIRST Lego League competitions every year. We've even helped to start robotics teams in rural Ecuador and help educate students living there. More information about our outreach efforts can be found <a href="outreach">here</a>.</p>
+        <p></p>
```
Comment (ErikBoesen) [inline index.html]: This extra paragraph just adds blank space, remove this.

File: index.html [hunk 8]
```diff
@@ -22,19 +22,20 @@
     <!-- TODO: Replace this header. "Welcome" doesn't make any sense. -->
     <h1 id="welcome">Welcome!</h1>
     <!-- TODO: Replace this tagline with something less narcissistic -->
-    <h3>We are one of the most successful and inventive FIRST Robotics teams in the Chesapeake District.</h3>
-    <a href="/robot/2017" class="button">View our 2017 Robot</a>
+    <h3>Woe To The Vanquished!</h3>
+    <!-- TODO: update this button, it is severly out of date -->
```
Comment (ErikBoesen) [inline index.html]: Not anymore...

File: index.html [hunk 9]
```diff
@@ -22,19 +22,20 @@
     <!-- TODO: Replace this header. "Welcome" doesn't make any sense. -->
     <h1 id="welcome">Welcome!</h1>
     <!-- TODO: Replace this tagline with something less narcissistic -->
-    <h3>We are one of the most successful and inventive FIRST Robotics teams in the Chesapeake District.</h3>
-    <a href="/robot/2017" class="button">View our 2017 Robot</a>
+    <h3>Woe To The Vanquished!</h3>
```
Comment (ErikBoesen) [inline index.html]: Keep the old subheading.

File: _pages/robot/2018.html [hunk 10]
```diff
@@ -1,44 +1,48 @@
...
-        <li><img src="/assets/img/first.svg">2017<br> Chesapeake Innovation in Control</li>
-    </ul>
-    <img src="/assets/img/robots/2017.jpg" alt="The 2017 robot being carried onto the ground.">
-    <p>The 2017 game is proving to be one of the most challenging to date for our experienced team. Designed by Disney, FIRST STEAMWORKS aims to challenge teams to create mechanisms capable of delivering 11 inch diameter gears to human players stationed on the Jules Verne inspired airship. Teams also have the task of shooting 5 inch diameter wiffle balls to add fuel to the boiler, sending steam to fuel the airship. For a more detailed description of the game, see the <a href="https://www.youtube.com/watch?v=EMiNmJW7enI">the official video reveal</a>.</p>
+    <h1>2018 Robot</h1>
+    <iframe src="https://www.youtube.com/v/jqxHeneuZwA"></iframe>
+    <!--TODO: add images-->
+    <p>The 2018 game's fusion of 20th century arcade gamming and 21st century robotics is possibly one of the most challenging scenario to date. For a more detailed description of the game, see the <a href="https://youtu.be/HZbdwYiCY74">the official video reveal</a>.</p>
```
Comment (ErikBoesen) [inline _pages/robot/2018.html]: scenarios, not scenario.

File: index.html [hunk 11]
```diff
@@ -23,18 +23,17 @@
...
-    <a href="/robot/2017" class="button">View our 2017 Robot</a>
+    <!-- TODO: update this button, it is severly out of date -->
+    <a href="/robot/2019" class="button">View our 2018 Robot</a>
 </div>
 <main>
     <div>
         <img src="/assets/img/team.jpg">
-        <p>We are team 1418 Vae Victis, a <a href="http://firstinspires.org/robotics/frc">FIRST Robotics Competition (FRC)</a> team from George Mason High School in Falls Church, VA. Since our founding in 2004, our team has grown in every way, and has maintained a high place in competition and in the community. To do this, we maintain an active and engaged student body of around fifteen students who work with the team on and off season.</p>
+        <p>We are team 1418 Vae Victis, a <a href="http://firstinspires.org/robotics/frc">FIRST Robotics Competition (FRC)</a> from George Mason High School in Falls Church, VA. Since our founding in 2004, our team has grown in every way, and has maintained a high place in competition and in the community. To do this, we maintain an active and engaged student body of around fifteen students who work with the team on and off season.</p>
```
Comment (ErikBoesen) [inline index.html]: Why did you remove the word "team?" This makes it nonsensical.

File: _pages/robot/2018.html [hunk 12]
```diff
@@ -7,8 +7,9 @@
 ---
 <main>
     <h1>2018 Robot</h1>
-    <iframe src="https://www.youtube.com/v/jqxHeneuZwA"></iframe>
-    <!--TODO: add images-->
+    <iframe src="/assets/img/REDONE.jpg" alt="Our current robot (front view)."></iframe>
```
Comment (ErikBoesen) [inline _pages/robot/2018.html]: `<iframe>`s are for videos. For images, you'll want to use `` tags.

**Comment by NathanielHorne** [comment]
Changes have been made. We need this page to go up. Please.
